src/plone/restapi/deserializer/site.py

```diff
@@ -10,68 +10,85 @@
 
 import json
 
+try:
+    from Products.CMFPlone.factory import PLONE60MARKER
+
+    PLONE60MARKER  # pyflakes
```

Comment:
Would this work? An extra line to make pyflakes happy is so ugly...

```suggestion
    from Products.CMFPlone.factory import PLONE60MARKER  # flake8: noqa
```

Reply:
I usually do:
```
from Products.CMFPlone.factory import PLONE60MARKER  # noqa: F401
```
but IMO trying to import stuff is not the correct way to get the Plone, I will come up with a suggestion

Reply:
I think I took it from somewhere else that was doing the same thing (with the PLONE52MARKER or so... I can change it indeed.